services/frontend-v3/src/components/careerInterests/CareerInterestsView.jsx

```diff
@@ -35,7 +35,7 @@ const CareerInterestsView = ({
         render: (
           <div style={{ marginTop: 7 }}>
             {relocationLocations.map(({ id, city, province }) => (
-              <Tag color="rgb(114, 114, 114)" key={id}>
+              <Tag color="#727272" key={id}>
```

Comment:
I think it is my fault that it is RGB there. sorry!
However, I think it was decided that only skill bubbles would be "#727272" the rest would remain gray. We can discuss it in the meeting today.

Reply:
#727272 is gray and is the same thing as rgb(114, 114, 114), I just changed it since we're not using rgb values in the app

Reply:
lol yikes! i copied #00605e into google and saw it was green and though i copied #727272. My bad!